packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  OP_CODES, Script, TransactionSegWit,
+  DestroyLoanScheme
+} from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+
+export class TxnBuilderLoans extends P2WPKHTxnBuilder {
+  /**
+   * Destroy a loan scheme. Currently requires Foundation Authorization.
+   *
+   * @param {DestroyLoanScheme} destroyLoanScheme txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+  async destroyLoanScheme (destroyLoanScheme: DestroyLoanScheme, changeScript: Script): Promise<TransactionSegWit> {
```

Comment:
https://github.com/DeFiCh/jellyfish/pull/594/files#r691771161 Inside this function.